crates/trigger/src/runtime_config.rs

```diff
@@ -198,8 +198,8 @@ pub struct RuntimeConfigOpts {
     #[serde(default)]
     pub llm_compute: Option<LlmComputeOpts>,
 
-    #[serde(rename = "config_provider", default)]
-    pub config_providers: Vec<ConfigProviderOpts>,
+    #[serde(rename = "variable_provider", default)]
```

Comment:
:bike::derelict_house::
```suggestion
    #[serde(rename = "variables_provider", default)]
```

Reply:
I originally did this but decided to drop the plurality because it started to sound funny in my head.

Reply:
I think it reads better to someone who hasn't recently gazed into the variables abyss.

Reply:
Ok i'm just going to undo all the switches from `variables*` to `variable*` that i made this morning. So going back to `VariablesProvider`.